src/initializers/diamorphosis.ts

```diff
@@ -93,6 +93,7 @@ export default (config, orkaOptions: Partial<OrkaOptions>) => {
     styles: [],
     headersRegex: '(^X-.*)|cf-ray',
     maxBodyValueChars: undefined,
+    hostFieldName: 'host',
```

Comment:
@nikostoulas I will use current host as the default value. Wdyt? 
In order to support DD all applications will have to set it in config for now that we are still in stackdriver production

Reply:
Makes sense. We should make a commit with SREs for all apps in k8s to add the respective env variable.

Reply:
As we will have to bump orka (and riviere) in all apps, we can also add it there in codebase